test/integration/util/util.go

```diff
@@ -95,33 +96,38 @@ func StartScheduler(clientSet clientset.Interface) (factory.Configurator, Shutdo
 		close(stopCh)
 		klog.Infof("destroyed scheduler")
 	}
-	return schedulerConfigurator, shutdownFunc
+	return sched.Config(), shutdownFunc
 }
 
-// createSchedulerConfigurator create a configurator for scheduler with given informer factory and default name.
-func createSchedulerConfigurator(
+// createSchedulerFactory create a factory for scheduler with given informer factory and default name.
+func createSchedulerFactory(
```

Comment:
This function returns a scheduler directly instead of a factory. So renaming it to `createScheduler`?

Reply:
SG